corehq/apps/domain/forms.py

```diff
@@ -2681,8 +2681,13 @@ class DomainAlertForm(forms.Form):
         required=False
     )
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, request, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+        datetime_local_widget_helptext = f"Using project's timezone: {request.project.default_timezone}"
```

Comment:
Should this be translated?

```suggestion
        datetime_local_widget_helptext = _("Using project's timezone: {}").format(
            request.project.default_timezone
        )
```